build.go

```diff
@@ -121,6 +125,25 @@ func Build(entryResolver EntryResolver, dependencyManager DependencyManager, env
 		logger.Action("Completed in %s", duration.Round(time.Millisecond))
 		logger.Break()
 
+		logger.GeneratingSBOM(nodeLayer.Path)
+		var sbomContent sbom.SBOM
+		duration, err = clock.Measure(func() error {
+			sbomContent, err = sbomGenerator.GenerateFromDependency(dependency, context.WorkingDir)
```

Comment:
```suggestion
			sbomContent, err = sbomGenerator.GenerateFromDependency(dependency, nodeLayer.Path)
```

I think this should be the path to the place on the filesystem where this dependency exists.